### ISSUE TITLE
Vulnerbility found in ws

### ISSUE THREAD

**Issue by fiskgrodan** [napcs/node-livereload#130]
## What version of Livereload are you using?

^0.7.0

Installed as a dependecy to **rollup-plugin-livereload**
https://github.com/thgh/rollup-plugin-livereload/blob/master/package.json

## What OS are you using?

macOS / GitHub

## Expected result

To not get a vulnerbility warning

## Actual result

Getting vulnerbility warnings in terminal and in the GitHub GUI.

This is the warning I get when installing:
```
+ rollup-plugin-livereload@1.0.0
added 241 packages from 138 contributors and audited 1219 packages in 5.31s
found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
```

I get these warnings in the GitHub GUI:
<img width="994" alt="Screenshot 2019-06-05 at 15 47 13" src="https://user-images.githubusercontent.com/26715792/58962255-1bf96f80-87ab-11e9-9ba9-4f6413f323a2.png">

<img width="770" alt="Screenshot 2019-06-05 at 15 47 32" src="https://user-images.githubusercontent.com/26715792/58962262-2156ba00-87ab-11e9-871b-c81d5226b342.png">

<img width="763" alt="Screenshot 2019-06-05 at 15 55 25" src="https://user-images.githubusercontent.com/26715792/58962274-2582d780-87ab-11e9-8f71-581b53060ba2.png">

## Steps to reproduce issue 

I installed it from this repo:
https://github.com/sveltejs/template

With these steps:
1. npx degit sveltejs/template svelte-app
2. npm install

## Why is this important?

Vulnerbility warnings give a bad impression and might scare of possible users. Maybe upgrading the **ws** module could fix the issue?

**Comment by napcs** [comment]
@fiskgrodan LiveReload v0.8.0 was released last week.

Running `npm audit` on Master currently shows no vulns:

```
                       === npm audit security report ===

found 0 vulnerabilities
 in 2233 scanned packages
```

I'll continue to monitor this.

**Comment by oliverxchen** [comment]
The commit linked to in the Github security report is: https://github.com/websockets/ws/commit/c4fe46608acd61fbf7397eadc47378903f95b78a

After creating the PR, I read down further into the comments and found this: https://github.com/websockets/ws/commit/c4fe46608acd61fbf7397eadc47378903f95b78a#commitcomment-28951427 which reports that 1.1.5 is not vulnerable. So maybe Github has it wrong?

**Comment by larafale** [comment]
same here, came from `rollup-plugin-livereload@1.0.1` which use `livereload@0.8.0`, and pushing my repo to github gave me a security alert (high), which can be resolved by upgrading ws to 3.3.1.

bummer, because running `npm audit` on both `livereload@master` & `rollup-plugin-livereload@1.0.1` and also my project return `found 0 vulnerabilities`

the joy of programming :)

**Comment by napcs** [comment]
Ok. I'll take everyone's word for it. Lemme make sure #131 resolves this properly and I'll package up a new release this week.

**Comment by larafale** [comment]
that would be awesome, thanks !

**Comment by larafale** [comment]
hello again @napcs, just a little reminder about that one ^^
I'll buy you a beer anytime :)
thanks, take care !

**Comment by napcs** [comment]
Closed by #131. 0.8.1 released

**Comment by larafale** [comment]
you've made my day :) thanks a lot
